### DISCUSSION TITLE
我想问问哪里有相对全面的组件属性？

### DISCUSSION THREAD

**Discussion by first19326** [walinejs/waline#1856] [Q&A] [answered]
我想问问哪里有相对全面的组件属性？比如说locales中的属性，我这不知道有哪些，得一个个的找。

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/reference/client/props.html

**Reply by first19326** [reply]
我知道这个，但是这个不全啊，里面没有 locale 属性的介绍，比如说 locale.admin 还有 locale.placeholder.我不确定是否还有这种配置，我知道这2个属性还是在 Github 以及别人页面 waline 配置里面找的。希望你可以理解。

**Reply by lizheming** [reply]
https://waline.js.org/reference/client/props.html 选到 lang -> 「功能->多语言」-> 「Cookbook → 自定义语言支持」能看到 locale 这是从我提供的页面怎么找到 locale，实际上直接在左侧目录上就可以看到「自定义语言」。文档自己多看看吧，这个文档就是最全的，locale 因为是和 lang 相关的，所以放在那了，其它的都在页面里。

**Reply by first19326** [reply]
非常感谢，是我之前没看到，抱歉。
